docs/admin/installation.md

```diff
@@ -12,10 +12,56 @@ modality: "universal"
 
 # Installation Guide
 
-This guide covers installing NeMo Curator and verifying your installation is working correctly.
+This guide covers installing NeMo Curator with support for **all modalities** and verifying your installation is working correctly.
 
 ## Before You Start
 
+### System Requirements
+
+For comprehensive system requirements and production deployment specifications, see [Production Deployment Requirements](deployment/requirements.md).
+
+**Quick Start Requirements:**
+
+- **OS**: Ubuntu 24.04/22.04/20.04 (recommended)
+- **Python**: 3.10, 3.11, or 3.12
+- **Memory**: 16GB+ RAM for basic text processing
+- **GPU** (optional): NVIDIA GPU with 16GB+ VRAM for acceleration
+
+### Install FFmpeg and Encoders
```

Comment:
`Install FFmpeg and Encoders` are not really required for other modalities, so I would say putting it after `Installation methods` might be better. Also maybe have a bracket like `Install FFmpeg and Encoders (Required only For Video)` @praateekmahajan what do you think ?